.gitignore

```diff
@@ -9,6 +9,8 @@
 *.cmxa
 *.log
 *.ll
+*.diff
+*.err
```

Comment:
needed for fail-test?

Reply:
I think these are the output of failing tests? Do you think we actually need them?

Reply:
Yes, we suppose fail-xxx.mb will fail and output the exact same error as stored in the fail-xxx.err files